rfc-report.jq

```diff
@@ -7,7 +7,8 @@ def has_label(l): .labels | any(.name == l);
   | length as $length
   | sort_by(.number)
   | [
-    { title: "Draft RFCs", items: map(select(has_label("status: draft"))) },
+    { title: "Draft RFCs", items: map(select(has_label("status: draft") and .draft)) },
+    { title: "Revived RFCs", items: map(select(has_label("status: draft") and (.draft | not))) },
```

Comment:
This is a bit weird to me. Why do we have a draft label and a draft status? Can we just remove the label and rely on the status?

Reply:
That would break exactly this method of detecting when an author wants to bring the RFC back to our attention. I think the labels are a good way of representing the "last status known to the steering committee".

Reply:
But if a RFC is no longer in draft status doesn't that bring it back to our attention on its own? The only difference I really see is that we see it in the regular non-draft section instead of having a separate "Revived" section.

Reply:
(some thinking later...) yes that makes sense.